impl/src/test/java/org/eclipse/parsson/JsonParserFixture.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package org.eclipse.parsson;
+
+import java.io.StringReader;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import jakarta.json.Json;
+import jakarta.json.JsonArray;
+import jakarta.json.JsonObject;
+import jakarta.json.stream.JsonParser;
+
+/**
+ * Class with methods that creates JsonParser with different configuration from different sources and runs the test code with this parser
+ */
+public class JsonParserFixture {
```

Comment:
That is a self-explanatory name :(

Reply:
Yes, I know. ;-) But what else should I write here? This is a fixture class for the tests involving the parser, so many of them. I don't like to repeat this try-with-resource line over and over again.

Or did you mean like we don't need the JavaDoc for the class itself, only the methods of this particular class? I mean, it's maybe not necessary, but this comment also doesn't hurt anybody, right? ;-)